styles.css

```diff
@@ -0,0 +1,87 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-position: 0.1rem 1.5cm;
+  background-repeat: no-repeat;
+  background-image: url(/img/background.svg);
+}
+
+a {
+  font-size: 12px;
+  letter-spacing: 1px;
+  text-decoration: none;
+  text-align: center;
+  display: inline-block;
+}
+
+.header ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  display: flex;/* - */
+  align-items: center;
+}
+
+.header li {
+  flex: 4;
+}
+
+.header a {
+  display: block;
+  width: 100%;
+  padding-left: 24px;
+  text-align: left;
+}
+
+.header .toggle-menu {
+  display: block;
+  flex: 1;
+  flex-grow: 1;
+  padding-right: 25px;
+  text-align: right;
+  align-self: center;
+}
+
+.content {
+  padding-top: 34px;
+  text-align: center;
+}
+
+.content .p1 {
+  font-size: 40px;
+  font-weight: bold;
+  line-height: 30px;
+}
```

Comment:
- [ ] Please add the correct font type and line-height because the given font type is `inter` and the correct line-height is `52px`.

![image](https://user-images.githubusercontent.com/68796232/185285190-14852131-243a-419a-8eee-cde8bb8e0864.png)

Reply:
Hello @Omar-Muhamad , I changed all the recommendations that you gave me but I couldn't change the font, I tried to import it but the styleint kept showing me an error. Do I have to ask for another review or you can review it here? Thank you.